README.md

```diff
@@ -1,159 +1,158 @@
 <div align="center">
-  <img src="assets/header.jpg"/><br>
-  <img src="https://github.com/tuist/tuist/workflows/Tuist/badge.svg" alt="Tuist">
-  <img src="https://img.shields.io/github/v/release/tuist/tuist?include_prereleases&style=flat-square" alt="Latest Version">
-  <img src="https://img.shields.io/github/commit-activity/w/tuist/tuist?style=flat-square" alt="Commit Activity">
-  <img src="https://img.shields.io/github/commits-since/tuist/tuist/latest?style=flat-square" alt="Latest Commits">
-  <a href="http://twitter.com/tuistio"><img src="https://img.shields.io/twitter/follow/tuistio?style=flat-square" alt="Twitter Followers"></a>
-  <a href="https://join.slack.com/t/tuistapp/shared_invite/zt-1lqw355mp-zElRwLeoZ2EQsgGEkyaFgg"><img src="https://img.shields.io/badge/slack-tuist-green.svg?logo=slack" alt="Slack Workspace"></a>
-  <img src="https://img.shields.io/opencollective/sponsors/tuistapp?style=flat-square" alt="Sponsors">
-  <img src="https://img.shields.io/opencollective/backers/tuistapp?style=flat-square" alt="Backers">
-  <img src="https://img.shields.io/github/license/tuist/tuist?style=flat-square" alt="License">
-  <img src="https://img.shields.io/badge/Powered%20by-Tuist-blue" alt="Powered by Tuist">
-  <a href="https://polar.sh/tuist"><img src="https://polar.sh/embed/seeks-funding-shield.svg?org=tuist" /></a>
-</div>
-
-<div align="center">
-  <a href="https://cal.com/team/tuist/cloud?utm_source=banner&utm_campaign=oss" target="_blank"><img alt="Book us with Cal.com" src="https://cal.com/book-with-cal-dark.svg" /></a>
```

Comment:
I personally don't like those badges and I think it's clearer without them, but it might be just my personal preference, let me know what you think

Reply:
I'd keep it because they provide a nice overview for someone coming across the repository for the first time:
- Activity in the repository
- Is CI green and therefore the people behind it ensure a healthy state of it, or is it read and that means they don't care about a healthy CI status?
- Where else can I find the people behind the project? They seem to have an X account and also a Slack channel that I can join.

Reply:
Added some of them back and fixed some stlying, let me know if you think we need more